test/runtests.jl

```diff
@@ -2,6 +2,7 @@ using SpeedyWeather
 using Test
 
 # GENERAL
+include("test_transforms_ad_rules.jl") # TODO: put the test somewhere else (and next to the transforms)
```

Comment:
Don't actually mind in which order the tests are executed but yeah feel free to move it after the transforms tests